docsrc/firebolt.client.rst

```diff
@@ -2,16 +2,22 @@
 Client
```

Comment:
Autogenerated piece for `UsernamePassword` and `Token` doesn't look properly as well, similarly to `firebolt.async_db.connect`